### PR TITLE
add subscription manager and readme update

### PR DESCRIPTION
**Pull request by leena-jawale**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,12 +90,39 @@ terraform plan -out=test-plan.tfplan
...
 
 ```bash
-terraform apply
+terraform apply -var infrastructure_db_password=<db-password> -var aap_admin_password=<aap-admin-password> -var aap_red_hat_username=<redhat-username> -var aap_red_hat_password=<redhat-password>
 ```
 Confirm to create infrastructure or pass in the `-auto-approve` parameter.
 
+### Installing Red Hat Ansible Automation Platform
+
+At this point you can ssh into one of the controller nodes and run the installer.
```
Comment (chrisahl) [inline README.md:99]: Please give an example command so the user knows how to do this and with which user name:  For example:
ssh azureuser@<controller public IP>

Comment (chrisahl) [reply README.md:99]: I see variable "infrastructure_admin_username" defaults to azureuser, so need to mention this too so they are aware of it.

**Review by chrisahl — COMMENTED** [review]
When I tried the first time, the subscription manager part did not seem to work and the inventory file did not have values populated for vm IPs and RHSM fields, etc.

I need to debug and try again

File: README.md [hunk 2]
```diff
@@ -28,12 +90,42 @@ terraform plan -out=test-plan.tfplan
...
+  ssh -i ~/.ssh/id_rsa azureuser@<controller-public-ip> 
+```
+We provided a sample inventory that could be used to deploy AAP.
+You might need to edit the inventory to fit your needs.
+
+Before you start the installation, you need to attach Ansible Automation Platform to the system where you're running the installer. 
+
+Find the pool id for Ansible Automation Platform subscription using command 
+```bash
+subscription-manager list --all --available
```
Comment (chrisahl) [inline README.md:110]: prefix with sudo

**Comment by chrisahl** [comment]
> When I tried the first time, the subscription manager part did not seem to work and the inventory file did not have values populated for vm IPs and RHSM fields, etc.
> 
> I need to debug and try again

I retried after the dependency was added and it worked much better!

**Comment by chrisahl** [comment]
After my azure terraform deploy worked, I ran the AAP installer on the controller and it finished successfully.  Then logged into controller UI
image


image

image

File: README.md [hunk 3]
```diff
@@ -2,11 +2,73 @@
 
 This is the template that will deploy Ansible on Azure. While this template will work with any Ansible deployment on Azure, this is intended to be a starting point for customers that purchase Ansible Automation Platform subscriptions from the Azure marketplace. Take this template and enhance/improve/update based on the resources that you need for your AAP deployment.
 
+## Introduction
+
+This template performs the following actions in the order listed.
+Some variables can be used to skip or prevent steps by setting variables, which are covered later in this document.
```
[the remark this answers is not in the document]
Comment (leena-jawale) [reply README.md]: Removed this line. I copy pasted most of the readme from ansible example. And In ansible example we can set var to skip creation of load balancer

File: README.md [hunk 4]
```diff
@@ -28,12 +90,44 @@ terraform plan -out=test-plan.tfplan
...
 
 ```bash
-terraform apply
+terraform apply -var infrastructure_db_password=<db-password> -var aap_admin_password=<aap-admin-password> -var aap_red_hat_username=<redhat-username> -var aap_red_hat_password=<redhat-password>
 ```
 Confirm to create infrastructure or pass in the `-auto-approve` parameter.
 
+### Installing Red Hat Ansible Automation Platform
+
+At this point you can ssh into one of the controller nodes and run the installer. The default `infrastructure_admin_username` is `azureuser`.
```
[the remark this answers is not in the document]
Comment (leena-jawale) [reply README.md]: updated

**Review by chrisahl — APPROVED** [review]

**Pull request merged**